src/gui.js

```diff
@@ -7614,6 +7632,9 @@ ProjectDialogMorph.prototype.saveProject = function () {
                 );
             } else {
                 this.ide.setProjectName(name);
+                location.hash =
```

Comment:
Side effect: If you open someone else's project and save to your account, the URL now updates to your project! 🎉